test/users.js

```diff
@@ -1,6 +1,7 @@
 const chai = require('chai'),
```

Comment:
Something I forgot to tell you in the previous PR, we try to imitate the folder structure of the app folder inside the tests. So this file should be in `/test/controllers/users.js`. Also we nest all the tests inside a describe with the name of the module we are testing. For example in this case should be something like:

```
describe('users controllers', () => {
  describe('/users POST', () => {
    ..
  });
  ..
});
```

If someone isn't clear enough let me know and we talk it